doc/nrf/includes/vsc_build_and_run.txt

```diff
@@ -0,0 +1,26 @@
+1. Open Visual Studio Code.
+
+   If you installed the |NCS| using the :ref:`gs_app_tcm`, you can click the :guilabel:`Open VS Code` button next to the version you installed.
+
+#. In |VSC|, open the :guilabel:`nRF Connect for VS Code` extension by clicking its icon or pressing ``Ctrl`` + ``Alt`` + ``N``.
```

Comment:
```suggestion
#. Open the |VSC| extension by clicking its icon or pressing ``Ctrl`` + ``Alt`` + ``N``.
```

I missed this one on the last iteration, and I don't think we need the first instance mentioning VSC because the step prior already says to open VSC.